music21/stream/base.py

```diff
@@ -494,6 +494,8 @@ def __getitem__(self,
         2
         >>> len(s[note.Note])
         6
+        >>> len(s[[note.Note, note.Rest]])
```

Comment:
This didn't work before. Now it works with or without the inner list brackets (i.e. it works with an implicit tuple from the multiple arguments to __getitem__)

Reply:
Better not to demo it with the additional inner list brackets (put in non-doc test) -- that looks really awkward, and the point of the syntax is to make m21 friendlier.